packages/data-types/src/command.ts

```diff
@@ -0,0 +1,165 @@
+import yargs from 'yargs';
+import path from 'path';
+import fs from 'fs';
+import { debugLogger, get } from '@terascope/utils';
+import { DataType, DataTypeConfig } from './index';
+
+// @ts-ignore
+const logger = debugLogger('ts-transform-cli');
```

Comment:
Don't copy and paste
```suggestion
const logger = debugLogger('ts-transform-cli');
```

Reply:
fixed